Bender.yml

```diff
@@ -13,7 +13,7 @@ package:
 
 dependencies:
   apb_uart:                 { git: "https://github.com/pulp-platform/apb_uart",               version: 0.2.1  }
-  axi:                      { git: "https://github.com/pulp-platform/axi",                    version: 0.39.0-beta.8 }
+  axi:                      { git: "https://github.com/pulp-platform/axi",                    rev: 5e85c7e    }
```

Comment:
Ideally, this should be a release; if that's not possible, at least a stable master commit before merging (in that case, let's indicate this with a TODO as for e.g. iDMA)

Reply:
Sure. This is WIP. I don't want to merge bc of this. But pressure is being applied.

Reply:
Done.

Reply:
We will use the existing AXI release for now, requiring an override for AXI RT to work. This allows us to make forward progress on other PRs without creating a master commit depending on a volatile AXI branch. See #40.